### PR TITLE
README.md: better usage instructions

### PR DESCRIPTION
**Pull request by Nowaker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,26 @@ Requirements
 -	[Terraform](https://www.terraform.io/downloads.html) 0.12.x
 -	[Go](https://golang.org/doc/install) 1.11 (to build the provider plugin)
 
+Using the Provider
```
Comment (mbag) [inline README.md:16]: Using the provider with Terraform 0.13 or higher

**Review by mbag — CHANGES_REQUESTED** [review]

**Comment by mbag** [comment]
hi @Nowaker  I commented about adding note that the syntax in README is for terraform v0.13 or higher. Once that is added I'll merge the PR. I don't expect someone to be using terraform <=0.12 but just so that it's clear with which version the syntax used works.

**Comment by Nowaker** [comment]
Done!

**Comment by c33s** [comment]
@mbag why the drop of older terraform versions? the provider runs quite fine with 0.11. and everything over 0.11 has a security impact for us. see https://github.com/hashicorp/terraform/issues/26141

**Comment by mbag** [comment]
@c33s  I'm actually not sure if older versions will be able to fetch newer releases directly from the Registry. I think that older versions are fetching providers from different URLs than registry.terraform.io.

**Comment by c33s** [comment]
@mbag it works, i manually downloaded the powerdns provider yesterday (updated from 1.3 to 1.4 with terraform 0.11) works like charm. having only the problems where i opened issues for. so 1.4 is 0.11 compatible

**Comment by mbag** [comment]
@c33s  manual download and placing provider in the search path of the `terraform` should work of course, that's why I wrote `directly from Registry`. Because Hashicorp has discontinued old way of releasing and only support publishing providers on the Registry.

**Pull request merged**